src/scss/components/_methodology.scss

```diff
@@ -0,0 +1,57 @@
+@use 'abstracts' as *;
+
+.methodology {
+  position: relative;
+  display: grid;
+  grid-column-gap: 72px;
+  grid-template-columns: 2fr 1fr;
+  padding-top: rem(60px);
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: #979797;
+  }
+
+
+  &__title {
+    grid-column: 1 / span 2;
```

Comment:
This is fine and doesn't need to change, but another way to right this is `grid-column: 1 / -1`, which means span all rows.

Reply:
So this actually does need to change now that we have the mobile version. Because it's specifying the spanning of two columns, it's actually creating a second column on mobile even when there isn't supposed to be one.

<img width="448" alt="Screen Shot 2019-11-04 at 12 18 09 PM" src="https://user-images.githubusercontent.com/1896496/68142296-3328be80-fefd-11e9-8be3-09958dea503e.png">